build alert using overide razor pages

I currently update abp from v2.8.0 to v3.0.4

I got an build alert 

```
9>------ 已启动生成: 项目: Yun.IdentityServer, 配置: Debug Any CPU ------
9>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2081,5): warning MSB3277: 发现“System.Diagnostics.PerformanceCounter”的不同版本间存在无法解决的冲突。当日志详细程度设置为“详细”时，这些引用冲突将会在生成日志中列出。
9>C:\Users\17449\.nuget\packages\microsoft.extensions.fileproviders.embedded\3.1.5\build\netstandard2.0\Microsoft.Extensions.FileProviders.Embedded.targets(19,5): warning : GenerateEmbeddedFilesManifest was set, but no EmbeddedResource items were found that could be added to the manifest.
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\LoggedOut.cshtml.g.cs(4,100,4,123): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\LoggedOut.cshtml.g.cs”中的类型“Pages_Account_LoggedOut”与“Volo.Abp.Account.Web.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages_Account_LoggedOut”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\LoggedOut.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Login.cshtml.g.cs(4,100,4,119): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Login.cshtml.g.cs”中的类型“Pages_Account_Login”与“Volo.Abp.Account.Web.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages_Account_Login”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Login.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Logout.cshtml.g.cs(4,100,4,120): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Logout.cshtml.g.cs”中的类型“Pages_Account_Logout”与“Volo.Abp.Account.Web.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages_Account_Logout”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Logout.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Manage.cshtml.g.cs(4,100,4,120): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Manage.cshtml.g.cs”中的类型“Pages_Account_Manage”与“Volo.Abp.Account.Web.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages_Account_Manage”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Manage.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Register.cshtml.g.cs(4,100,4,122): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Register.cshtml.g.cs”中的类型“Pages_Account_Register”与“Volo.Abp.Account.Web.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages_Account_Register”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\Register.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\_Layout.cshtml.g.cs(4,100,4,121): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\_Layout.cshtml.g.cs”中的类型“Pages_Account__Layout”与“Volo.Abp.Account.Web.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages_Account__Layout”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\_Layout.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\_ViewStart.cshtml.g.cs(4,100,4,124): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\_ViewStart.cshtml.g.cs”中的类型“Pages_Account__ViewStart”与“Volo.Abp.Account.Web.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages_Account__ViewStart”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\Account\_ViewStart.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\_ViewImports.cshtml.g.cs(4,100,4,118): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\_ViewImports.cshtml.g.cs”中的类型“Pages__ViewImports”与“Volo.Abp.Account.Web.IdentityServer.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Pages__ViewImports”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Pages\_ViewImports.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Brand\Default.cshtml.g.cs(4,100,4,137): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Brand\Default.cshtml.g.cs”中的类型“Themes_Basic_Components_Brand_Default”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_Brand_Default”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Brand\Default.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\MainNavbar\Default.cshtml.g.cs(4,100,4,142): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\MainNavbar\Default.cshtml.g.cs”中的类型“Themes_Basic_Components_MainNavbar_Default”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_MainNavbar_Default”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\MainNavbar\Default.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Menu\Default.cshtml.g.cs(4,100,4,136): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Menu\Default.cshtml.g.cs”中的类型“Themes_Basic_Components_Menu_Default”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_Menu_Default”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Menu\Default.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Menu\_MenuItem.cshtml.g.cs(4,100,4,138): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Menu\_MenuItem.cshtml.g.cs”中的类型“Themes_Basic_Components_Menu__MenuItem”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_Menu__MenuItem”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Menu\_MenuItem.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\PageAlerts\Default.cshtml.g.cs(4,100,4,142): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\PageAlerts\Default.cshtml.g.cs”中的类型“Themes_Basic_Components_PageAlerts_Default”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_PageAlerts_Default”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\PageAlerts\Default.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\Default.cshtml.g.cs(4,100,4,139): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\Default.cshtml.g.cs”中的类型“Themes_Basic_Components_Toolbar_Default”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_Toolbar_Default”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\Default.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\LanguageSwitch\Default.cshtml.g.cs(4,100,4,154): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\LanguageSwitch\Default.cshtml.g.cs”中的类型“Themes_Basic_Components_Toolbar_LanguageSwitch_Default”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_Toolbar_LanguageSwitch_Default”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\LanguageSwitch\Default.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\UserMenu\Default.cshtml.g.cs(4,100,4,148): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\UserMenu\Default.cshtml.g.cs”中的类型“Themes_Basic_Components_Toolbar_UserMenu_Default”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Components_Toolbar_UserMenu_Default”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Components\Toolbar\UserMenu\Default.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Account.cshtml.g.cs(4,100,4,128): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Account.cshtml.g.cs”中的类型“Themes_Basic_Layouts_Account”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Layouts_Account”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Account.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Application.cshtml.g.cs(4,100,4,132): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Application.cshtml.g.cs”中的类型“Themes_Basic_Layouts_Application”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Layouts_Application”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Application.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Empty.cshtml.g.cs(4,100,4,126): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Empty.cshtml.g.cs”中的类型“Themes_Basic_Layouts_Empty”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic_Layouts_Empty”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\Layouts\Empty.cshtml.g.cs”中定义的类型。
9>D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\_ViewImports.cshtml.g.cs(4,100,4,125): warning CS0436: “D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\_ViewImports.cshtml.g.cs”中的类型“Themes_Basic__ViewImports”与“Volo.Abp.AspNetCore.Mvc.UI.Theme.Basic.Views, Version=3.0.4.0, Culture=neutral, PublicKeyToken=null”中的导入类型“Themes_Basic__ViewImports”冲突。请使用“D:\source\repos\YbMicroServices\applications\Yun.IdentityServer\obj\Debug\netcoreapp3.1\Razor\Themes\Basic\_ViewImports.cshtml.g.cs”中定义的类型。

```

How can I clear this alert

--- a/common.props
+++ b/common.props
@@ -2,7 +2,7 @@
   <PropertyGroup> 
     <LangVersion>latest</LangVersion> 
     <Version>3.2.0</Version>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0436</NoWarn>
     <PackageIconUrl>https://abp.io/assets/abp_nupkg.png</PackageIconUrl>
     <PackageProjectUrl>https://abp.io/</PackageProjectUrl>
     <PackageLicenseFile>LICENSE.md</PackageLicenseFile>

--- a/common.test.props
+++ b/common.test.props
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0436</NoWarn>
     <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>

--- a/modules/cms-kit/common.props
+++ b/modules/cms-kit/common.props
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <Version>0.1.0</Version>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0436</NoWarn>
 	<AbpProjectType>module</AbpProjectType>
   </PropertyGroup>
 

--- a/modules/identityserver/common.props
+++ b/modules/identityserver/common.props
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <Version>0.3.0</Version>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0436</NoWarn>
     <PackageIconUrl>http://www.aspnetboilerplate.com/images/abp_nupkg.png</PackageIconUrl>
     <PackageProjectUrl>http://abp.io</PackageProjectUrl>
     <RepositoryType>git</RepositoryType>

--- a/templates/app/aspnet-core/common.props
+++ b/templates/app/aspnet-core/common.props
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <Version>1.0.0</Version>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0436</NoWarn>
 	<AbpProjectType>app</AbpProjectType>
   </PropertyGroup>
 </Project>

--- a/templates/console/common.props
+++ b/templates/console/common.props
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <Version>0.1.0</Version>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0436</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>

--- a/templates/module/aspnet-core/common.props
+++ b/templates/module/aspnet-core/common.props
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
     <Version>0.1.0</Version>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn);CS1591;CS0436</NoWarn>
 	<AbpProjectType>module</AbpProjectType>
   </PropertyGroup>
 

